Sources/GRPC/TLSVerificationHandler.swift

```diff
@@ -37,8 +37,8 @@ public enum GRPCApplicationProtocolIdentifier: String, CaseIterable {
 /// Users of this handler should rely on the `verification` future held by this instance.
 ///
 /// On fulfillment of the promise this handler is removed from the channel pipeline.
-public class TLSVerificationHandler: ChannelInboundHandler, RemovableChannelHandler {
-  public typealias InboundIn = Any
+internal class TLSVerificationHandler: ChannelInboundHandler, RemovableChannelHandler {
+  internal typealias InboundIn = Any
```

Comment:
Given that the entire class is `internal`, shall we drop that modifier on the internal fields and methods?

Reply:
I prefer the explicit `internal` but fine changing if you prefer?

Reply:
Not feeling too strongly, but it does feel quite redundant to me if the same visibility is already specified on the enclosing scope. I think Swift even warns for these redundant visibilities in some cases.

Reply:
Haven't seen the warnings but let's remove them anyway.